WcaOnRails/app/controllers/api/v0/api_controller.rb

```diff
@@ -28,11 +28,11 @@ def auth_results
   def scramble_program
     render json: {
       "current" => {
-        "name" => "TNoodle-WCA-0.15.0",
+        "name" => "TNoodle-WCA-0.15.1",
         "information" => "#{root_url}regulations/scrambles/",
-        "download" => "#{root_url}regulations/scrambles/tnoodle/TNoodle-WCA-0.15.0.jar",
+        "download" => "#{root_url}regulations/scrambles/tnoodle/TNoodle-WCA-0.15.1.jar",
       },
-      "allowed" => ["TNoodle-WCA-0.15.0"],
+      "allowed" => ["TNoodle-WCA-0.15.1"],
```

Comment:
To allow for the "usual week to phase out the old version" that @lgarron mentioned, this actually needs to be updated to allow for both versions.

Reply:
I don't think this was the case since one of the version suddenly became against regs. Continuing via email.